configs/efficientdet/README.md

```diff
@@ -8,9 +8,9 @@ Main features of EfficientDet models are EfficientNet backbone and BiFPN neck.
 ## Results and Models
 |     Backbone    |    Head    |  Style  | Lr schd | Mem (GB) | Inf time (fps) | box AP | Download |
 | :-------------: | :--------: | :-----: | :-----: | :------: | :------------: | :----: | :------: |
-| efficientnet_b0 | [retinahead](./retinanet_effd0_bifpn_1x_coco.py) | pytorch |  300e   |   22.0   | 33.3           |  30.2  | [model](https://storage.openvinotoolkit.org/repositories/mmdetection/models/efficientdet/retinanet_effd0_bifpn_1x_coco/epoch_300.pth) |
+| efficientnet_b0 | [retinahead](./retinanet_effd0_bifpn_1x_coco.py) | pytorch |  300e   |   14.2   | 40.9           |  30.0  | [model](https://storage.openvinotoolkit.org/repositories/mmdetection/models/efficientdet/retinanet_effd0_bifpn_1x_coco/epoch_300.pth) |
```

Comment:
Are reference metrics for tests still the same as before?

Reply:
New reference metrics have been pushed up, waiting on checkpoint to be uploaded to run tests